pages/book/operators.mdx

```diff
@@ -347,46 +409,80 @@ nullable == anotherNullable; // false
+
+Can only be applied to values of type [`Int{:tact}`][int]:
 
 ```tact
 let two: Int = 2;
-two > 2; // false
--1 > -3; // true
+two & 1;          // 0
+ 4  & 1;          // 0
+ 3  & 1;          // 1
+ 1  & 1;          // 1
+
+255        & 0b00001111; // 15
+0b11110000 & 0b00001111; // 15
 ```
 
-#### Greater than or equal to, `>=` [#binary-greater-equal]
+<Callout>
 
-Binary _greater than or equal to_ operator `>={:tact}` returns `true{:tact}` if the left operand is greater than or to the right operand, and `false{:tact}` otherwise. Can only be applied to values of type [`Int{:tact}`][int]:
+  [Bitwise AND - Wikipedia](https://en.wikipedia.org/wiki/Bitwise_operation#AND)\
+  [Bit manipulation - Wikipedia](https://en.wikipedia.org/wiki/Bit_manipulation)
 
-```tact
-let two: Int = 2;
-two >= 2; // true
--1 >= -3; // true
-```
+</Callout>
 
-#### Less than, `<` [#binary-less]
+### Bitwise XOR, `^` [#binary-bitwise-xor]
 
-Binary _less than_ operator `<{:tact}` returns `true{:tact}` if the left operand is less than the right operand, and `false{:tact}` otherwise. Can only be applied to values of type [`Int{:tact}`][int]:
+Binary caret (_bitwise XOR_) operator `^{:tact}` applies a [bitwise XOR](https://en.wikipedia.org/wiki/Bitwise_operation#XOR), which performs the [logical exclusive OR](https://en.wikipedia.org/wiki/Exclusive_or) operation on each pair of the corresponding bits of operands. The result in each position is $1$ if only one of the bits is $1$, but will be $0$ if both are $0$ or both are $1$. In this it performs the comparison of two bits, giving $1$ if the two bits are different, and $0$ if they are the same.
+
+It is useful for inverting selected bits of an operand (also called toggle or flip), as any bit may be toggled by "XORing" it with $1$.
+
+Can only be applied to values of type [`Int{:tact}`][int]:
 
 ```tact
 let two: Int = 2;
-two < 2; // false
--1 < -3; // false
+two ^ 3; // 1
+ 4  ^ 1; // 0
+ 3  ^ 1; // 3
+ 1  ^ 1; // 0
+
+255        ^ 0b00001111; // 240
+0b11110000 ^ 0b00001111; // 240
 ```
 
-#### Less than or equal to, `<=` [#binary-less-equal]
+<Callout>
 
-Binary _less than or equal to_ operator `<={:tact}` returns `true{:tact}` if the left operand is less than or equal to the right operand, and `false{:tact}` otherwise. Can only be applied to values of type [`Int{:tact}`][int]:
+  [Bitwise XOR - Wikipedia](https://en.wikipedia.org/wiki/Bitwise_operation#XOR)\
+  [Bit manipulation - Wikipedia](https://en.wikipedia.org/wiki/Bit_manipulation)
+
+</Callout>
+
+### Bitwise OR, `|` [#binary-bitwise-or]
+
+Binary bar (_bitwise OR_) operator `|{:tact}` applies a [bitwise OR](https://en.wikipedia.org/wiki/Bitwise_operation#OR), which performs the [logical OR](#binary-logical-or) operation on each pair of the corresponding bits of operands. This is useful when we want to apply a specific [bitmask](https://en.wikipedia.org/wiki/Mask_(computing)).
```

Comment:
Speaking of bit masks, here would be a nice place to link to the sendmodes combinations section